code/game/objects/items/weapons/landmines.dm

```diff
@@ -0,0 +1,146 @@
+/obj/item/weapon/landmine
+	name = "land mine"
+	desc = "An anti-personnel explosive device used for area denial."
+	icon = 'icons/obj/grenade.dmi'
+	icon_state = "landmine"
+	throwforce = 0
+	w_class = 3
+	var/deployed = 0
+
+/obj/item/weapon/landmine/update_icon()
+	..()
+
+	if(!deployed)
+		icon_state = "[icon_state]"
+	else
+		icon_state = "[icon_state]_on"
+
+/obj/item/weapon/landmine/verb/hide_under()
+	set src in oview(1)
+	set name = "Hide"
+	set category = "Object"
+
+	if(use_check(user, USE_DISALLOW_SILICONS))
+		return
+
+	layer = TURF_LAYER+0.2
+	usr << "<span class='notice'>You hide \the [src].</span>"
+
+
+/obj/item/weapon/landmine/attack_self(mob/user as mob)
+	..()
+	if(!deployed && use_check(user, USE_DISALLOW_SILICONS))
```

Comment:
This is probably the opposite you want, `use_check()` returns `FALSE` on success.